envi/tests/test_arch_amd64.py

```diff
@@ -1081,7 +1107,7 @@ def test_envi_amd64_disasm_Reg_Operands(self):
         opbytez = '0440'
         oprepr = 'add al,64'
         opcheck = {'iflags': 131072, 'prefixes': 0, 'mnem': 'add', 'opcode': 8193, 'size': 2}
-        opercheck = ( {'tsize': 4, 'reg': 524288}, {'tsize': 1, 'imm': 64} )
+        opercheck = ( {'tsize': 4, 'reg': 524288}, {'tsize': 4, 'imm': 64} )
```

Comment:
?  what's the question?  if it's supposed to be 1, what changed?

Reply:
I think I got it. It's part the table was missing a bit, but I left that alone in favor of making it so that register operands embedded in opcodes (so addrmeth_0) overrode their tsize's

Reply:
but don't we *want* it to be 1?

Reply:
Yup. We do. I fixed that in my last commit. Both of them are 1, just like the should be, since al is an 8 bit register.